File: trail1/비교 연산/comparison-operator.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var (A, B) = (int.Parse(arr[0]), int.Parse(arr[1]));
        bool[] lst = {(A >= B), (A > B), (B >= A), (B > A), (A == B), (A != B)};
        for (int i=0; i<6; i++) {
            if (lst[i]) {Console.WriteLine(1);}
            else {Console.WriteLine(0);}
        }
    }
}

File: trail1/비교 연산/comparison-operator.py
lst = list(map(int, input().split()))
A, B = lst[0], lst[1]
arr = [(A >= B), (A > B), (B >= A), (B > A), (A == B), (A != B)]
result = [1 if i else 0 for i in arr]
print(*result, sep="\n")